code/_onclick/hud/movable_screen_objects.dm

```diff
@@ -89,6 +92,34 @@
 	else if(findtext(Y,"CENTER"))
 		. = usr.client.view+1
 
+// STORAGE
+/obj/screen/movable/storage
+	name = "storage"
+	layer = 19
+	screen_loc = "4:16,2:16"
+	mouse_opacity = 2
+
+/obj/screen/movable/storage/MouseDrag(over_object, src_location, over_location, src_control, over_control, params)
+	..()
+	if(!movable)
+		return
+	if(istype(master, /obj/item/weapon/storage/))
+		var/obj/item/weapon/storage/S = master
+		S.space_orient_objs()
+
+/obj/screen/movable/storage/Click()
+	if(!usr.canClick())
+		return 1
+	if(usr.stat || usr.paralysis || usr.stunned || usr.weakened)
```

Comment:
`use_check()` with specific flags might be a better idea.

Reply:
```suggestion
	if(use_check(usr, USE_ALLOW_NON_ADV_TOOL_USR))
```
Monkeys et al. should still be able to remove things from bags.